images/scripts/rhel4edge.bootstrap

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+set -eux
+
+BASE=$(dirname $0)
+
+# Names needs to be unique
+timestamp=$(date +%s)
+request=$(cat <<EOF
+{
+  "name": "rhel4edge-$timestamp",
+  "version": 0,
+  "distribution": "rhel-91",
+  "imageType": "rhel-edge-installer",
+  "packages": [
+    { "name": "cockpit-system" },
+    { "name": "cockpit-bridge" },
+    { "name": "socat" },
+    { "name": "openssh-server" }
+  ],
+  "outputTypes": [
+    "rhel-edge-installer",
+    "rhel-edge-commit"
+  ],
+  "commit": {
+    "arch": "x86_64"
+  },
+  "installer": {
+    "username": "admin",
```

Comment:
can we set the password right here as well? and put it into `wheel`?

Reply:
well, this actually does nothing :man_shrugging: 
I am not sure if I understand this correctly, but this is "installer" part, so maybe it is not for the image itself? But in any case, it does not create such user. I contacted edge team about it. But I cannot drop it because it is required.

Reply:
Opened issue https://issues.redhat.com/browse/THEEDGE-3018  and also mentioned it as hack in `images/scripts/rhel4edge.setup`